chemprop/args.py

```diff
@@ -383,6 +385,22 @@ class TrainArgs(CommonArgs):
     """Overwrites the default atom descriptors with the new ones instead of concatenating them"""
     no_bond_features_scaling: bool = False
     """Turn off atom feature scaling."""
+    frzn_mpnn: bool = False
```

Comment:
Can we collapse this argument into other arguments?

As set up now, mpnn is frozen any time anything is frozen. And we need a checkpoint_frzn any time anything is frozen. So we could just key off of whether there is a checkpoint_frzn to decide if we are freezing the mpnn.

If we had situations where mpnn was not frozen but ffn was, then you would need this extra degree of freedom. But we don't have it currently configured in a way right now where it's possible since the frzn_ffn will override a false on frzn_mpnn. I don't think that's an option that we actually need in the future, so I think we're safe to remove this argument.